modules/infotext/clock.lua

```diff
@@ -0,0 +1,284 @@
+-- Clock Infotext
+
+-- ####################################################################################################################
+-- ##### Setup and Locals #############################################################################################
+-- ####################################################################################################################
+
+local _, LUI = ...
+local module = LUI:GetModule("Infotext")
+local element = module:NewElement("Clock", "AceEvent-3.0", "AceHook-3.0")
+local L = LUI.L
+
+-- local copies
+local gsub, format = gsub, format
+local tonumber, date = tonumber, date
+local GetNumSavedWorldBosses = GetNumSavedWorldBosses
+local GetSavedWorldBossInfo = GetSavedWorldBossInfo
+local GetSavedInstanceInfo = GetSavedInstanceInfo
+local GetNumSavedInstances = GetNumSavedInstances
+local TimeBreakDown = ChatFrame_TimeBreakDown
+local GetInstanceInfo = GetInstanceInfo
+local IsInInstance = IsInInstance
+local GetGameTime = GetGameTime
+
+-- local variables
+local cvarLocal, cvarMilitary -- Cache containing CVars
+local guildParty         -- If there's a G or not in the text
+local instanceInfo       -- Any instance tag would go in this.
+local invitesPending = false
+
+-- constants
+local GAMETIME_TOOLTIP_TOGGLE_CALENDAR = GAMETIME_TOOLTIP_TOGGLE_CALENDAR
+local TIMEMANAGER_TOOLTIP_LOCALTIME = TIMEMANAGER_TOOLTIP_LOCALTIME
+local TIMEMANAGER_TOOLTIP_REALMTIME = TIMEMANAGER_TOOLTIP_REALMTIME
+local TIMEMANAGER_TITLE = TIMEMANAGER_TITLE
+local TIMEMANAGER_PM = TIMEMANAGER_PM
+local TIMEMANAGER_AM = TIMEMANAGER_AM
```

Comment:
Would you find appending `_G.` to most of the globals functions/constants here (and in the other files) ?
ie: `local TIMEMANAGER_PM = _G.TIMEMANAGER_PM`

Don't have luacheck configured in GitHub, but it's a simple way to be explicit about them being global variables.